baipw/tests/test_utils.py

```diff
@@ -43,39 +43,3 @@ def test_authorise_with_invalid_header(self):
         self.assertEqual(
             str(e.exception), "Invalid format of the authorization header."
         )
-
-
-class TestGetClientIP(TestCase):
-    def setUp(self):
-        self.request = RequestFactory().get("/")
-
-    def test_get_client_ip_from_remote_addr(self):
-        self.request.META["REMOTE_ADDR"] = "192.168.0.17"
-        self.assertNotIn("HTTP_X_FORWARDED_FOR", self.request.META)
-        self.assertIn("REMOTE_ADDR", self.request.META)
-        self.assertEqual(get_client_ip(self.request), "192.168.0.17")
-
-    def test_get_client_ip_if_no_remote_addr_or_x_forwaded_for(self):
-        del self.request.META["REMOTE_ADDR"]
-        self.assertNotIn("HTTP_X_FORWARDED_FOR", self.request.META)
-        self.assertNotIn("REMOTE_ADDR", self.request.META)
-        self.assertIsNone(get_client_ip(self.request))
-
-    def test_get_client_ip_from_x_forwaded_for(self):
-        self.request.META["HTTP_X_FORWARDED_FOR"] = "72.123.123.89"
-        self.assertIn("HTTP_X_FORWARDED_FOR", self.request.META)
-        self.assertIn("REMOTE_ADDR", self.request.META)
-        self.assertEqual(get_client_ip(self.request), "72.123.123.89")
-
-    def test_get_client_ip_from_x_forwaded_for_when_multiple_values(self):
-        self.request.META["HTTP_X_FORWARDED_FOR"] = "72.123.123.89,5.123.2.45"
-        self.assertIn("HTTP_X_FORWARDED_FOR", self.request.META)
-        self.assertIn("REMOTE_ADDR", self.request.META)
-        # Should use the last IP from the list.
-        self.assertEqual(get_client_ip(self.request), "5.123.2.45")
-
-    def test_get_client_ip_prioritises_cloudflare_ip(self):
-        self.request.META["HTTP_CF_CONNECTING_IP"] = "72.123.123.90"
-        self.request.META["HTTP_X_FORWARDED_FOR"] = "110.123.123.89"
-        self.assertIn("REMOTE_ADDR", self.request.META)
-        self.assertEqual(get_client_ip(self.request), "72.123.123.90")
```

Comment:
**Suggestion:** If you are going to delete this, I would add tests that mock ipware to make sure it gets called and passes through the value. This may be not relevant if we agree on the other comments.